extension/src/plots/util.test.ts

```diff
@@ -0,0 +1,58 @@
+import { sep } from 'path'
+import { standardisePlotsDataPaths } from './util'
+import { PlotsType } from './webview/contract'
+import { FIELD_SEPARATOR } from '../cli/dvc/constants'
+import { PlotsOutput } from '../cli/dvc/contract'
+
+const join = (pathArr: string[], slash = sep) => pathArr.join(slash)
```

Comment:
[N] Naming this `join` threw me off, I thought `join(['plots', 'heatmap.png'], slash)` was using some magical undocumented path for `path.join`